packages/manager/apps/web/client/app/dns-zone/new/dns-zone-new.html

```diff
@@ -1,313 +1,137 @@
-<div class="page-header mb-5">
-    <h1>
-        <span data-translate="domains_newdnszone_order_title"></span>
-        <small
-            class="d-block"
-            data-translate="domains_newdnszone_order_subtitle"
-        ></small>
-    </h1>
-</div>
+<oui-page-header
+    class="mb-3"
+    heading="{{:: 'domains_newdnszone_order_title' | translate }}"
+    description="{{:: 'domains_newdnszone_order_subtitle' | translate }}"
+>
+    <oui-guide-menu data-text="{{:: 'guide_domain_subtitle' | translate }}">
+        <oui-guide-menu-item href="{{ $ctrl.GUIDE_URL }}" external>
+            <span data-translate="domains_newdnszone_order_help_add"></span>
+        </oui-guide-menu-item>
+    </oui-guide-menu>
+</oui-page-header>
 
-<div class="container-fluid px-5">
-    <div class="row">
-        <div class="col-md-9">
-            <div data-ovh-alert="{{ctrlNewDnsZone.alerts.main}}"></div>
+<div class="px-5">
+    <div data-ovh-alert="{{$ctrl.alerts.main}}"></div>
+    <p data-translate="domains_newdnszone_order_example"></p>
+    <p class="font-italic" data-translate="domains_newdnszone_order_usage"></p>
 
-            <p data-translate="domains_newdnszone_order_example"></p>
-            <p
-                class="font-italic"
-                data-translate="domains_newdnszone_order_usage"
-            ></p>
-            <div
-                class="alert alert-warning mt-5"
-                role="alert"
-                data-translate="domains_newdnszone_order_reminder"
-            ></div>
+    <oui-message data-type="warning">
+        <span data-translate="domains_newdnszone_order_reminder"></span>
+    </oui-message>
 
-            <!-- step 1 -->
-            <h2
-                class="mt-5"
-                data-translate="domains_newdnszone_order_step1_title"
-            ></h2>
-            <form name="newDnsZoneForm">
+    <ovh-manager-product-offers
+        workflow-type="'order'"
```

Comment:
You should use this constant: https://github.com/ovh/manager/blob/master/packages/manager/modules/product-offers/src/workflows/product-offers-workflow.constants.js#L7